spec/requests/api/v1/articles/index_spec.rb

```diff
@@ -1,16 +1,18 @@
 RSpec.describe 'GET /api/v1/articles', type: :request do
-  let(:headers) { { HTTP_ACCEPT: 'application/json' } }
+  let(:journalist) { create(:user, role: 'journalist')}
+  let!(:journalist_headers) { { HTTP_ACCEPT: 'application/json' } }
   let!(:article) do
     10.times do
       create(:article,
             title: 'Breaking News',
-            body: 'Some breaking content'*10)
+            body: 'Some breaking content'*10,
+            journalist_id: journalist.id)
```

Comment:
Why do you feel you need to do this here rather than letting the factory take care of assigning the journalist association?

Reply:
I left ```journalist_id: journalist.id``` as is to use one journalist id for all ten articles, otherwise the 10.times create function tries to create 10 journalist with the 10 articles and sends email in use errors. (i think this is the cause of the errors). Tried a few work arounds unsuccessfully.

Reply:
@EevanR we do need a journalist here so please change it back to `:headers`, remove the line where you create the user,  and remove the journalist id from the factory create.